packages/main/types/function-calling.ts

```diff
@@ -0,0 +1,163 @@
+import { FunctionCallingMode } from "./enums";
```

Comment:
copyright.

Reply:
FYI `yarn format` handles this automatically so we don't need to check for it in reviews.